ouroboros-consensus/src/Ouroboros/Consensus/Fragment/Validated.hs

```diff
@@ -10,11 +11,10 @@
 -- > import Ouroboros.Consensus.Fragment.Validated (ValidatedFragment)
 -- > import qualified Ouroboros.Consensus.Fragment.Validated as VF
 module Ouroboros.Consensus.Fragment.Validated (
-    ValidatedFragment -- Opaque
+    ValidatedFragment(ValidatedFragment)
```

Comment:
Why this change? So you can pattern match on it instead of using the accessors?

Reply:
Yup, exactly.